src/components/Theme/utils.ts

```diff
@@ -12,6 +13,14 @@ export const setUICustomizations = (customUI: UICustomizationOptions): void => {
   }
 }
 
+export const setCobrandingLogos = (
+  logoCobrandConfig: EnterpriseLogoCobranding
+): void => {
+  const style = document.createElement('style')
+  style.textContent = `:root { --darkLogoSrc: url(${logoCobrandConfig.darkLogoSrc}); \n --lightLogoSrc: url(${logoCobrandConfig.lightLogoSrc});}`
```

Comment:
I think the CSS variable name needs to be less generic, something like `--osdk-dark-cobrand-logo-src` and `--osdk-light-cobrand-logo-src` (kebabcase for consistency with the UI customisation CSS variables' convention). On the off chance the client has a `--darkLogoSrc`, `--lightLogoSrc`.

Reply:
Updated